include/glow/ExecutionContext/TraceEvents.h

```diff
@@ -265,7 +278,13 @@ class ScopedTraceBlock {
   bool end_{false};
 
 public:
-  ScopedTraceBlock(TraceContext *context, llvm::StringRef name);
+  /// TraceContext constructor.
+  ScopedTraceBlock(TraceContext *context, TraceLevel level,
+                   llvm::StringRef name);
+
+  /// ExecutionContext constructor.
```

Comment:
typo

Reply:
I'd just remove those comments, since it does not bring any value. nbd anyway.

Reply:
What's the typo here too? I don't see it.

I can remove, though.

Reply:
`ExecutionContext constructor.` like this is not ExecutionContext constructor.

Reply:
Ah! It's the constructor that *takes* an `ExecutionContext`, as opposed to the constructor that takes a `TraceContext`. I could make that clearer or I could just delete both, what do you think?

Reply:
I'd just delete :)